packages/cli/commands/functions/deploy.js

```diff
@@ -71,26 +77,22 @@ exports.handler = async options => {
   );
 
   try {
-    spinner = ora(
-      i18n(`${i18nKey}.loading`, {
-        accountId,
-        functionPath,
-      })
-    ).start();
     const buildId = await buildPackage(accountId, functionPath);
     const successResp = await pollBuildStatus(accountId, buildId);
     const buildTimeSeconds = (successResp.buildTime / 1000).toFixed(2);
-    spinner.stop();
+    spinnies.remove('buildAndDeployStatus');
+    spinnies.stopAll();
```

Comment:
I could be mistaken but I don't think you need `stopAll()` if the spinner has been removed

Reply:
By no means a blocker, but the other thing I'm wondering is whether we should use the `succeed()` or `fail()` methods that spinnies provides to potentially improve the UX. Haven't given much thought to that, but figured I'd mention it now that we have new functionality available

Reply:
@anthmatic, I had to do a bit of research to uncover this workflow, but if I simply use the `remove()` method without `stopAll()`, the CLI command hangs forever. The `stopAll()` command allows the command to finish in the absence of `succeed()` or `fail()`. 

I wasn't sure whether we wanted to continue to use `logger` for messaging, but I love your suggestion to use `succeed()` or `fail()` in place of the clunkier `remove()` and `stopAll()` methods. Hopping on it now.

Reply:
@anthmatic, coming back with an update on this.

I tried to replace our `logger.success` and `logger.error` messages with `spinnies.succeed` and `spinnies.fail`, but I ran into some issues. 

In the `api-sample` command, the initial message `Loading available API samples` is visible for two steps where the customer first chooses the API sample app and second the app's language. We need to stop the spinnie before the customer chooses this, but there's no way to stop a spinnie without removing it entirely (please tell me if I'm wrong, but I did read through the docs pretty extensively and looked through sample code). 

In the `functions deploy` command, while using spinnies, I always hit the `else` command [here](https://github.com/HubSpot/hubspot-cli/blob/1c181ac74129869a377caea1f299990e6f96ce49/packages/cli/commands/functions/deploy.js#L111), even when I succeed in deploying the function. 

Having experimented with this for a bit now, I would suggest that we keep the code as is.